recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
     sha256: 7716d6ea20b950e71e8535faa4353ac89716c03fd7a445dd802eb6a630796639
 
 build:
-    number: 1002
+    number: 2002
```

Comment:
Was this already migrated?

cc @conda-forge/core

Reply:
Looks like it